src/ocm/components/clusterConfiguration/DiscoveryImageModal.tsx

```diff
@@ -69,22 +88,30 @@ export const DiscoveryImageModal = () => {
       hasNoBodyWrapper
       id="generate-discovery-iso-modal"
     >
-      {isoDownloadError && <ErrorState />}
+      {(isoDownloadError || ipxeDownloadError) && <ErrorState />}
       {isoDownloadUrl ? (
-        <DiscoveryImageSummary
-          clusterName={cluster.name || ''}
+        <DownloadIso
+          fileName={`discovery_image_${cluster.name || ''}.iso`}
```

Comment:
We should put back the suffix depending on the CPU architecture. It can also be used for the ipxe component.